### PR TITLE
Custom heads

### PR DESCRIPTION
**Pull request by codeallthethingz**
15799e31-cd98-4e87-9d49-40ceb4eb439e

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,23 +1,27 @@
 test:
-	go test -timeout 20s -race -coverprofile coverage.txt -covermode=atomic ./...
```
Comment (dlsteuer) [inline Makefile]: would be good to keep -race this will check data race conditions, especially since you are using channels

Comment (codeallthethingz) [reply Makefile]: it's adding 7 or 8 seconds to each run, should I separate this out into some other make instruction?

Comment (dlsteuer) [reply Makefile]: i guess you could, then setup travis to run the race detector

**Review by dlsteuer — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (codeallthethingz) [reply Makefile]: added it back in.  I can live with 9 seconds. :)

**Pull request merged**